packages/suite/src/components/wallet/CoinjoinSummary/CoinJoinCustomStrategy.tsx

```diff
@@ -0,0 +1,211 @@
+import { Button, P, Switch, variables } from '@trezor/components';
+import { CoinjoinSessionDetail } from './CoinjoinSessionDetail';
+
+const Row = styled.div`
+    display: flex;
+    justify-content: space-between;
+`;
+
+const TopRow = styled(Row)`
+    align-items: center;
+    margin-bottom: 20px;
+`;
+
+const MiddleRow = styled(Row)`
+    gap: 24px;
+    margin-bottom: 8px;
+`;
+
+const BottomRow = styled(Row)`
+    gap: 12px;
+    margin-top: 16px;
+`;
+
+const Heading = styled.div`
+    font-size: ${variables.FONT_SIZE.H3};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const SetupHeading = styled(Heading)`
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+`;
+
+const Fee = styled(Heading)`
+    color: ${({ theme }) => theme.TYPE_GREEN};
+`;
+
+const Subheading = styled.div`
+    font-size: ${variables.FONT_SIZE.NORMAL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    margin-bottom: 3px;
+`;
+
+const Text = styled(P)`
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const MiningFeeText = styled(Text)`
+    margin-top: 12px;
+`;
+
+const LeftMark = styled(Text)`
+    flex: 1 1 33%;
+`;
+
+const MiddleMark = styled(Text)`
+    flex: 1 1 33.5%;
+    text-align: center;
+`;
+
+const RightMark = styled(Text)`
+    flex: 1 1 33.5%;
+    text-align: right;
+`;
+
+const MiningFee = styled.section`
+    flex-grow: 1;
+`;
+
+const sliderTrack = css`
+    background: linear-gradient(
+        270deg,
+        #bf6767 0%,
+        #c8b882 18.73%,
+        #c8b883 36.25%,
+        #95cda5 43.99%,
+        #2a9649 100%
+    );
+    cursor: pointer;
+    height: 3px;
+`;
+const sliderThumb = css`
+    appearance: none;
+    background: #ffffff;
+    border: ${({ theme }) => `1px solid ${theme.STROKE_GREY}`};
+    border-radius: 50%;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2);
+    cursor: grab;
+    height: 26px;
+    margin-top: -12px;
+    width: 26px;
+
+    :active {
+        box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.2);
+        cursor: grabbing;
+    }
+`;
+
+const Range = styled.input`
+    appearance: none;
+    margin: 30px 0 12px 0;
+    vertical-align: top; /* prevent extra bottom space in Firefox */
+    width: 100%;
+
+    ::-webkit-slider-runnable-track {
+        ${sliderTrack}
+    }
+    ::-webkit-slider-thumb {
+        ${sliderThumb}
+    }
+    ::-moz-range-track {
+        ${sliderTrack}
+    }
+    ::-moz-range-thumb {
+        ${sliderThumb}
+    }
+`;
+
+const DetailWrapper = styled.section`
+    background: ${({ theme }) => theme.BG_GREY};
+    border-radius: 4px;
+    flex-basis: 35%;
+    padding: 16px;
+`;
+
+const DetailHeading = styled.div`
+    font-size: ${variables.FONT_SIZE.TINY};
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    margin-bottom: 16px;
+    text-transform: uppercase;
+`;
+
+const StyledSwitch = styled(Switch)`
+    margin-top: 2px;
```

Comment:
On Figma it's 10px, since it's move optically balanced that way ;)

```suggestion
    margin-top: 10px;
```

Reply:
3770dcd2737b99b21f4c2d835c606a32271e4214